src/main/java/com/dataloom/datastore/edm/controllers/EdmController.java

```diff
@@ -143,15 +146,28 @@ private void updateEntitySetIdsToGet(
             Set<UUID> propertyTypeIds,
             Set<UUID> entityTypeIds,
             Set<UUID> entitySetIds ) {
+        boolean setRetrieved = false;
```

Comment:
sure; just the quickest way I can think of to avoid doing IMap.get whenever possible - probably bad practice.